pkgs/tools/bluetooth/blueman/default.nix

```diff
@@ -31,10 +31,11 @@ in stdenv.mkDerivation rec {
 
   configureFlags = [ (lib.enableFeature withPulseAudio "pulseaudio") ];
 
-  preFixup = ''
+  postFixup = ''
```

Comment:
Because this package includes wrapGAppsHooks. If you wrap in preFixup, then wrapGAppsHooks wraps both the executable and its wrapper. This gives the insane wrappers chain described in #28260.

Now that I write it, I wonder why the same does not occur with wrapPythonProgramsIn. Maybe because it only wraps python packages, not wrappers ?